src/app/modules/summary/summary.controller.js

```diff
@@ -59,6 +60,36 @@ angular
        */
       vm.roundID = $routeParams.roundID;
 
+      vm.saveIDRoundPlayer();
+      /**
+       * @ngdoc method
+       * @name SummaryController#saveIDRoundPlayer
+       * @kind function
+       * @methodOf app.summary.controller:SummaryController
+       * @description
+       * Save the last roundID and playerID
+       */
+      function saveIDRoundPlayer() {
+        //Save the last Round ID
```

Comment:
Perche salvare l'id? non ti basta salvare la partita? in automatico getNextPlayer dovrebbe essere in grado gia di sapere chi è il prossimo giocatore no?!

Reply:
Posso usare getNextPlayer anche per la funzione loadMatch  (al posto di "localStorage.getItem('savePlayerID')" ) in players.controller.js?

Reply:
se ha senso farlo perche no?! quale è il dubbio?

Reply:
Probabilmente sbaglio qualcosa, ma se faccio così e aggiorno la pagina o la riapro, non perdo le informazioni? E così facendo getNextPlayer mi restituisce il 1° giocatore selezionato, e non l'ultimo che stava giocando.

Reply:
`getNextPlayer` deve restituire il primo giocatore (disponibile) che non ha completato il turno in teoria... Tutti i dati sono salvati in local storage. Hai gia verificato la tua tesi?

Reply:
Si esatto, solo che se non inizializzo il valore "currentPlayer", la funzione getCurrentPlayer ritorna falsa, e quindi getNextPlayer mi restituisce il 1° giocatore selezionato, dovrei inizializzare "currentPlayer" quindi, ma come faccio se non sò quale è l'ultimo giocatore che ha completato il turno? Dove sbaglio?